plugins/woocommerce/src/Internal/Api/Autogenerated/GraphQLQueries/GetCoupon.php

```
@@ -0,0 +1,53 @@
+				),
+				'code' => array(
+					'type'         => Type::string(),
+					'description'  => __( 'The coupon code to look up.', 'woocommerce' ),
+					'defaultValue' => null,
+				),
+			),
+			'resolve'     => array( self::class, 'resolve' ),
+		);
+	}
+
+	public static function resolve( mixed $root, array $args, mixed $context, ResolveInfo $info ): mixed {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$root'. (undefined)

(UnusedFormalParameter)